mackup/applications/sequel-ace.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = Sequel Ace
+
+[configuration_files]
+Library/Containers/com.sequel-ace.sequel-ace/Data/Library/Application Support/Sequel Ace/Data
+Library/Containers/com.sequel-ace.sequel-ace/Data/Library/Preferences/com.sequel-ace.sequel-ace.plist
```

Comment:
newline missing, will fix it.

Reply:
Sorry about the newline. I don't have any experience with Library/Containers. If you need to close the pull request, please do so. Sorry for that and my lack of experience...

Reply:
I can't test it, do you use this mackup configuration and does it work?